com.unity.render-pipelines.core/ShaderLibrary/Texture.hlsl

```diff
@@ -92,6 +92,8 @@ UnityTexture2D UnityBuildTexture2DStructInternal(TEXTURE2D_PARAM(tex, samplersta
     return result;
 }
 
+float2 GetTransformedUV(UnityTexture2D tex, float2 uv)      { return uv * tex.scaleTranslate.xy + tex.scaleTranslate.zw; }
```

Comment:
Could we put this as a member function on UnityTexture2D?    Keeps it out of the global namespace so we don't have to worry about collisions.